apps/github-cascading-app/src/cascading/interfaces.ts

```diff
@@ -34,7 +36,10 @@ export interface PullRequestContext {
   currentBranch: string;
   /** Cascading Pull Request Target Branch */
   targetBranch: string;
-  /** Determine if the reviewers are bypassed */
+  /**
+   * Determine if the reviewers are bypassed
+   * Note: This option is not supported on Github anymore due to Github Api change.
```

Comment:
Why are not we deprecating it ?

Reply:
I did not deprecate this option because I think that as the cascading is already today platform agnostic (event if it is implemented only for Github) and this option still make sense for supporting platform, it would be weird to remove it in 1 year to follow the unique implementation to potentially seeing it reappear after a while because of a new implementation.
I did make more sense for me to keep it and warn the Github users instead, but I am OK to change if needed.

PS: To implement another platform support, there is no need to extract code, it's the bot that should switch the implementation according to the received message. So today it is already ready to support multi platform as it is